pkg/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"crypto/sha512"
+	"fmt"
+)
+
+const MaxChainLength = 29 - len("CNI-")
```

Comment:
This is max name of name, not the whole chain. The var name is misleading. Also, shouldn't we take into account `-` twice?